Sources/Confidence/ConfidenceValue.swift

```diff
@@ -2,16 +2,171 @@ import Foundation
 
 public typealias ConfidenceStruct = [String: ConfidenceValue]
 
+public class ConfidenceValue: Equatable, Encodable {
+    private let value: ConfidenceValueInternal
+
+    public init(boolean: Bool) {
+        self.value = .boolean(boolean)
+    }
+
+    public init(string: String) {
+        self.value = .string(string)
+    }
+
+    public init(integer: Int64) {
+        self.value = .integer(integer)
+    }
+
+    public init(double: Double) {
+        self.value = .double(double)
+    }
+
+    public init(date: DateComponents) {
+        self.value = .date(date)
+    }
+
+    public init(timestamp: Date) {
+        self.value = .timestamp(timestamp)
+    }
+
+    // TODO: Handle heterogeneous types
+    public init(valueList: [ConfidenceValue]) {
+        self.value = .list(valueList.map { $0.value })
+    }
```

Comment:
Here we could make this private (or delete this), but then the problem is how to convert heterogeneous Values from OpenFeature

Reply:
oh, you had that just below :D